formapp/app.py

```diff
@@ -48,6 +48,11 @@
 import time, math
 import threading
 import logging
+
+#This import allows for the time to be printed to console. If you need to add a new print line the 2nd line is an example of how to use the logFile() function to make it print to console and to the log file
```

Comment:
We don't generally comment on import files, unless it's a small bit on the same line. Be concise!